main.go

```diff
@@ -0,0 +1,532 @@
+	if err != nil {
+		return xcodeproj.XcodeProj{}, "", err
+	}
+
+	return project, scheme.Name, nil
+}
+
+// buildTargetDirForScheme returns the TARGET_BUILD_DIR for the provided scheme
+func buildTargetDirForScheme(proj xcodeproj.XcodeProj, projectPath, scheme, configuration, simulatorPlatform string) (string, error) {
+	// Fetch project's main target from .xcodeproject
+	simulatorName := iOSSimName
+	if simulatorPlatform == "tvOS" {
+		simulatorName = tvOSSimName
+	}
+
+	var buildSettings serialized.Object
+	if xcodeproj.IsXcodeProj(projectPath) {
+		mainTarget, err := mainTargetOfScheme(proj, scheme)
+		if err != nil {
+			return "", fmt.Errorf("failed to fetch project's targets, error: %s", err)
+		}
+
+		buildSettings, err = proj.ProjectBuildSettings(mainTarget.Name, configuration, simulatorName)
+		if err != nil {
+			return "", fmt.Errorf("failed to parse project (%s) build settings, error: %s", projectPath, err)
+		}
+	} else if xcworkspace.IsWorkspace(projectPath) {
+		workspace, err := xcworkspace.Open(projectPath)
+		if err != nil {
+			return "", fmt.Errorf("Failed to open xcworkspace (%s), error: %s", projectPath, err)
+		}
+
+		buildSettings, err = workspace.WrokspaceBuildSettings(scheme, configuration, simulatorName)
+		if err != nil {
+			return "", fmt.Errorf("failed to parse workspace (%s) build settings, error: %s", projectPath, err)
+		}
+	} else {
+		return "", fmt.Errorf("project file extension should be .xcodeproj or .xcworkspace, but got: %s", filepath.Ext(projectPath))
+
+	}
+
+	schemeBuildDir, err := buildSettings.String("TARGET_BUILD_DIR")
+	if err != nil {
+		return "", fmt.Errorf("failed to parse build settings, error: %s", err)
+	}
+
+	return schemeBuildDir, nil
+}
+
+// exportArtifacts exports the main target and it's .app dependencies.
+func exportArtifacts(proj xcodeproj.XcodeProj, scheme string, schemeBuildDir string, configuration, XcodebuildOptions, simulatorPlatform, deployDir string) ([]string, error) {
+	var exportedArtifacts []string
+	splitSchemeDir := strings.Split(schemeBuildDir, "Build/")
+	if len(splitSchemeDir) != 2 {
+		return nil, fmt.Errorf("failed to parse scheme's build target dir: %s", schemeBuildDir)
+	}
+
+	mainTarget, err := mainTargetOfScheme(proj, scheme)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch project's targets, error: %s", err)
+	}
+
+	targets := append([]xcodeproj.Target{mainTarget}, mainTarget.DependentTargets()...)
+
+	for _, target := range targets {
+		log.Donef(target.Name + "...")
+
+		// Is the target an application? -> If not skip the export
+		if !strings.HasSuffix(target.ProductReference.Path, ".app") {
+			log.Printf("Target (%s) is not an .app - SKIP", target.Name)
+			continue
+		}
+
+		//
+		// Find out the sdk for the target
+		simulatorName := iOSSimName
+		if simulatorPlatform == "tvOS" {
+			simulatorName = tvOSSimName
+		}
+		{
+			sdkRoot, err := target.BuildConfigurationList.BuildConfigurations[0].BuildSettings.Value("SDKROOT")
```

Comment:
use project/workspace.ShowBuildSettings

Reply:
Sadly, it is not possible.
ShowBuildSettings need the sdk parameter, which is have not figured out yet.